components/tooltip/tooltip.js

```diff
@@ -991,6 +991,13 @@ class Tooltip extends RtlMixin(LitElement) {
 		target.appendChild(cloneContainer);
 		await this.updateComplete;
 
+		// if the clone is a web component it needs to update to fill in any slots
+		const customElm = customElements.get(clone.localName);
+		if (customElm !== undefined) {
+			clone.requestUpdate();
+			await clone.updateComplete;
+		}
+
```

Comment:
@NicholasHallman Are you seeing that this is only a problem because we're focusing on the link too quickly in the test? Or doesn't matter how long we wait, this is still needed?  Just wondering if the new `fixture` the vdiffs use would fix it, which wait for all components inside to update.

Reply:
I thought that might be the case but it wasn't working in the d2l-tooltip demo either. The element wouldn't contain anything inside the d2l-link so it had 0 width and never thought it needed to be truncated